problemtools/verifyproblem.py

```diff
@@ -644,6 +656,35 @@ def get_config(self):
         return ret
 
 
+class Attachments(ProblemAspect):
+    def __init__(self, problem):
+        self._problem = problem
+        attachments_path = os.path.join(problem.probdir, 'attachments')
+        if os.path.isdir(attachments_path):
+            self.attachments = os.listdir(attachments_path)
+        else:
+            self.attachments = []
```

Comment:
It seems like every time the attachments field is used, it is joined with the path of the problem to get an absolute path.  With that in mind it would probably make sense to store full paths in the attachments field instead?